JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -128,7 +129,7 @@ private async Task<bool> CallMessageHandler(Message message)
             var handlerSucceeded = await handler(message).ConfigureAwait(false);
 
             watch.Stop();
-            _log.LogTrace($"Handled message - MessageType: {message.GetType()}");
+            _logger.LogTrace("Handled message {MessageType}", message.GetType().ToString());
```

Comment:
To string is redundant here, it will do that anyway I believe.

Reply:
_”Handled message of type {MessageType}.”_

Reply:
Could add the time too. _”Handled message of type {MessageType} in {TimeToHandle}.”_